src/netaddress.h

```diff
@@ -109,8 +109,9 @@ class CNetAddr
         bool IsRFC4843() const; // IPv6 ORCHID (deprecated) (2001:10::/28)
         bool IsRFC7343() const; // IPv6 ORCHIDv2 (2001:20::/28)
         bool IsRFC4862() const; // IPv6 autoconfig (FE80::/64)
-        bool IsRFC6052() const; // IPv6 well-known prefix (64:FF9B::/96)
-        bool IsRFC6145() const; // IPv6 IPv4-translated address (::FFFF:0:0:0/96)
+        bool IsRFC6052() const; // IPv6 well-known prefix for IPv4-embedded address (64:FF9B::/96)
+        bool IsRFC6145() const; // IPv6 IPv4-translated address (::FFFF:0:0:0/96) (actually defined in RFC2765)
+        bool IsHeNet() const;   // IPv6 Hurricane Electric - https://he.net (2001:0470::/36)
```

Comment:
This adds more than upsteam did https://github.com/bitcoin/bitcoin/pull/17812/files only adds IsHeNet

Reply:
Aside from modifying comments surrounding `IsRFC6052()` and `IsRFC6145()`, it only adds `IsHeNet()`, would you like those modified comments removed?

Reply:
True true :+1: 

Modified comments are fine I guess, although try to avoid that in future